packages/paste-website/src/pages/components/button/index.mdx

```diff
@@ -193,6 +193,17 @@ possible, reversible action without much cost.
 </Button>`}
 </LivePreview>
 
+#### Inverse button
+
+The inverse button is best used over very strong colored backgrounds. In a lighter theme, this would mean over a dark background.
+Do reach for the other variants first and only use the inverse as a backup option.
+
+<LivePreview scope={{Button}} language="jsx">
+  {`<Button variant="inverse">
```

Comment:
Just need to wrap this is a Box with an inverse background so we can see the exaple.